ui/app/components/clients/charts/line.ts

```diff
@@ -132,7 +132,7 @@ export default class LineChart extends Component<Args> {
   };
   // These functions are used by the tooltip
   formatCount = (count: number) => {
-    return formatNumbers([count]);
+    return numericalAxisLabel([count]);
```

Comment:
Why are we sending this number as an array? Looks like `numericalAxisLabel` takes a number